app/docker/rest/system.js

```diff
@@ -1,16 +1,16 @@
 angular.module('portainer.docker')
-.factory('System', ['$resource', 'API_ENDPOINT_ENDPOINTS', 'EndpointProvider', function SystemFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider) {
+.factory('System', ['$resource', 'API_ENDPOINT_ENDPOINTS', 'EndpointProvider', 'InfoInterceptor', 'VersionInterceptor',
+  function SystemFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider, InfoInterceptor, VersionInterceptor) {
   'use strict';
   return $resource(API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/:action/:subAction', {
     name: '@name',
     endpointId: EndpointProvider.endpointID
   },
   {
     info: {
-      method: 'GET', params: { action: 'info' },
-      ignoreLoadingBar: true
+      method: 'GET', params: { action: 'info' }, timeout: 10000, interceptor: InfoInterceptor
     },
-    version: { method: 'GET', params: { action: 'version' }, ignoreLoadingBar: true, timeout: 4500 },
+    version: { method: 'GET', params: { action: 'version' }, timeout: 4500, interceptor: VersionInterceptor },
```

Comment:
Any particular reason to remove `ignoreLoadingBar: true` here?

Reply:
`Home` view does 3 API calls before going to `Dashboard` : `_ping`, `version` and `info`.
If we ignore the loading bar for `version`, you will have it for `ping`, then nothing for `version`, then have it back for `info`.
When the user is waiting for the `version` call to come back when browsing offline without loading bar, it's not UX friendly, as he will think the app is not working (nothing indicates the app is waiting for a response).